cmake/modules/FindDPDK.cmake

```diff
@@ -69,6 +69,27 @@ if(DPDK_FOUND)
     message("Version: ${DPDK_VERSION}")
     message("-----------")
   endif()
+
+  find_package_handle_standard_args(
```

Comment:
this code is common between pkg-config and a find search no? Why duplicating it ?